### PR TITLE
Fix multiple items for douban.com

### PR DESCRIPTION
**Pull request by JanusChoi**
The original Xpath ```//div/a[contains(@onclick, "moreurl")]``` is not working and I replace it with ```//div[@class="title"]/a'``` and modified following code in order to import multiple items in doulist or search result pages.

### CODE REVIEW

File: Douban.js [hunk 1]
```diff
@@ -205,13 +202,26 @@ function detectWeb(doc, url) {
 
 function doWeb(doc, url) {
 	var articles = new Array();
-	if (detectWeb(doc, url) == "multiple") { 
+	Zotero.debug(url);
+	if(detectWeb(doc, url) == "multiple") { 
 		var items = {};
-		var titles = doc.evaluate('//div/h2/a[contains(@onclick, "moreurl")]', doc, null, XPathResult.ANY_TYPE, null);
+		//var titles = doc.evaluate('//div/a[contains(@onclick, "moreurl")]', doc, null, XPathResult.ANY_TYPE, null);
+		Zotero.debug("Outputing title !!!")
```
Comment (zuphilip) [inline Douban.js]: Delete the comments and debug messages here and elsevier.

File: Douban.js [hunk 2]
```diff
@@ -205,13 +202,26 @@ function detectWeb(doc, url) {
 
 function doWeb(doc, url) {
 	var articles = new Array();
-	if (detectWeb(doc, url) == "multiple") { 
+	Zotero.debug(url);
+	if(detectWeb(doc, url) == "multiple") {
```
Comment (zuphilip) [inline Douban.js]: Let us keep the space after the `if` for better readability.

File: Douban.js [hunk 3]
```diff
@@ -205,13 +202,26 @@ function detectWeb(doc, url) {
...
+			Zotero.debug(title.href);
+			Zotero.debug(title.textContent);
 			items[title.href] = title.textContent;
 		}
+		//title = titles.iterateNext();
+		//Zotero.debug(title);
+		//while (title = titles.iterateNext()) {
+			//Zotero.debug(title);
+			//items[title.href] = title.textContent;
+		//}
```
Comment (zuphilip) [inline Douban.js]: Delete these lines.

File: Douban.js [hunk 4]
```diff
@@ -15,17 +15,14 @@
...
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.
-
    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.
-
    You should have received a copy of the GNU General Public License
```
Comment (zuphilip) [inline Douban.js]: Don't delete these lines, also they are blank.

**Comment by zuphilip** [comment]
Can you also add 1-2 test cases please or at least indicate some urls for testing?

**Comment by JanusChoi** [comment]
> Did you made sure that the test cases are still working as before? This is easy to test when you use Scaffold plugin within Zotero.

Thanks for the review!

Yes, I've already tested two links from the test cases and they work fine in Scaffold.

**Comment by zuphilip** [comment]
Thank you very much @JanusChoi ! Merged now :tada:

**Pull request merged**